airflow/executors/kubernetes_executor.py

```diff
@@ -459,8 +459,8 @@ def run_next(self, next_job: KubernetesJobType) -> None:
         if isinstance(command, str):
             command = [command]
```

Comment:
Hmm. L465:463 makes this str -> List conversion pointless i.e we no longer support strings

Reply:
Yeah, that's an _old_ workaround, it should go too.

Reply:
Done in https://github.com/apache/airflow/pull/9240/commits/0c817a5b4370f12117e59a82b5858f8ee6d488ff